### PR TITLE
Documentation: src/job.js

### PR DESCRIPTION
**Pull request by nahuakang**
## Description
This PR addresses [Issue #125](https://github.com/OpenMined/syft.js/issues/125) and makes edits or additions to `src/job.js`'s inline documentation.

Note that the inline documentation will be continuously improved as I progress through the repository :)

## Affected Dependencies
N/A

## How has this been tested?
N/A

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [ ] My changes are covered by tests

### CODE REVIEW

**Comment by cereallarceny** [comment]
@vvmnnnkv this is up to you to review.

@nahuakang Are you ready to take this out of WIP or do you need more time?

**Comment by nahuakang** [comment]
@cereallarceny I'll make it `ready for review` later tonight or tomorrow. Will notify Vova once it's ready!

File: src/job.js [hunk 1]
```diff
@@ -1,230 +1,232 @@
...
+       * `error` event.
+       * Triggered for plethora of error conditions.
+       *
+       * @event Job#error
+       */
+      this.observer.broadcast('error', error);
+      return;
+    }
+
+    // Trigger events for acacepted or rejected outside the try/catch.
```
Comment (vvmnnnkv) [inline src/job.js]: Typo :)

File: src/events.js [hunk 2]
```diff
@@ -1,18 +1,40 @@
+/**
+ * EventObserver is used by Syft, Job, and GridAPIClient to subscribe, unsubscribe or broadcast events.
```
Comment (vvmnnnkv) [inline src/events.js]: I think we don't need to list all classes that use EventObserver, at it's likely to change :)

**Review by vvmnnnkv — APPROVED** [review]

**Pull request merged**